packages/meteor/dynamics_browser.js

```diff
@@ -40,25 +40,10 @@ EVp.withValue = function (value, func) {
   try {
     currentValues[this.slot] = value;
 
-    var ret = func();
```

Comment:
Please don't forget to add a comment explaining why this should not behave the same as the server and how it interacts with methods and stubs. So another newbie doesn't make the same mistake 😄

Reply:
Also, I think this breaks some tests I either added or modified.